### DISCUSSION TITLE
Typeable date component

### DISCUSSION THREAD

**Discussion by thiagoalgo** [tasteui/tasteui#488] [Ideas]
In some cases, such as when the field is used to enter the date of birth, some users prefer to type the date instead of selecting it from the calendar. Is there any intention and forecast to develop this feature?

**Comment by devajmeireles** [comment]
I think we can go ahead with this. It seems logical.

**Reply by thiagoalgo** [reply]
Thanks, AJ. I'll wait for the development of this feature.

**Reply by MGeurts** [reply]
I second this. This is a much in demand request.
